stubs/networkx/@tests/stubtest_allowlist.txt

```diff
@@ -30,3 +30,20 @@ networkx\.algorithms\.bipartite\.(cluster\.)?clustering
 #   failing to account for explicitly passing in the default value."
 # Which is true, but would require some way of concatenating `backend` to ParamSpec.kwargs
 networkx\.(utils\.)?(backends\.)?_dispatchable\.__call__
+
+# Tests are excluded
+networkx.conftest
+networkx(\..+?)?\.tests(\..+?)?
+
+# Stubtest crashes due to special unicode character
+networkx.readwrite.text.*
+
+# "..._DT is not present at runtime" but we don't set it in stubs, I don't understand this one
+networkx(\.algorithms)?(\.tree)?(\.mst)?\.SpanningTreeIterator\.Partition\._DT
+networkx(\.algorithms)?(\.tree)?(\.branchings)?\.ArborescenceIterator\.Partition\._DT
+
+# variable differs from runtime type abc.ABCMeta
+networkx.classes.reportviews.EdgeView.dataview
+networkx.classes.reportviews.InEdgeView.dataview
+networkx.classes.reportviews.OutEdgeView.dataview
+networkx.classes.reportviews.OutMultiEdgeView.dataview
```

Comment:
These two issues I don't understand. I'm not sure if it's something I should do differently, just ignore, and/or report to mypy/stubtest